app/views/users/show.html.erb

```diff
@@ -1,2 +1,44 @@
-<h1>Users</h1>
-<p>Shows info for a the user</p>
+<section>
+  <div >
+    <div >
+      <img src="<%= @user.photo %>" alt="image">
+    </div>
+    <div>
+    <h2>
+    <%= @user.name %></h2>
+      <p>
+      <span>Number of posts: </span>
+      <%= @user.posts_counter || 0 %>
+      </p>
+
+  </div>
+</div>
+
+<div>
+  <h4> Bio </h4>
+  <p>
+    <%= @user.bio %>
+  </p>
+</div>
+
+<div>
+  <% @user.recent_posts.each do |post|  %>
+  <%= link_to user_post_path(@user, id: post:id) do %>
+    <p> Post <% post.id %></p>
+    <p><%= post.text %></p>
+    <p> 
+      <span>
+      Comments: <%= post.comments_counter || 0 %>,
+      Likes: <%= post.likes_counter || 0 %>
+      </span>
+    </p>
+  <% end %>
```

Comment:
- [ ] Here I am getting this error, Kindly resolve this.
![Screenshot from 2023-02-04 19-38-14](https://user-images.githubusercontent.com/26767383/216772084-3eca7da5-47e0-4ca9-8b03-dcb87648bd01.png)